modules/microvm/virtualization/microvm/appvm.nix

```diff
@@ -116,6 +117,13 @@ let
               nixpkgs.buildPlatform.system = configHost.nixpkgs.buildPlatform.system;
               nixpkgs.hostPlatform.system = configHost.nixpkgs.hostPlatform.system;
 
+              environment.extraInit = ''
```

Comment:
Isn't this imported with common?

Reply:
Yes, yes it is. Thanks, forgot to remove this.